app/components/support_interface/application_summary_component.rb

```diff
@@ -119,6 +120,17 @@ def average_distance_row
       }
     end
 
+    def editable_until_row
+      {
+        key: 'Editable until',
+        value: application_form.editable_until? ? application_form.editable_until.to_fs(:govuk_date_and_time) : nil,
```

Comment:
Feels like this row shuld be presented differenly, as it looks a bit confusing to me. Something like 'Candidate can edit aplication:` `No`, `Yes, until [date]`. Thoughts @DFE-Digital/find-and-apply-designers ?

Reply:
Happy to change :). Let me know what to change ✌️ 😁

Reply:
Needs some feedback from @DFE-Digital/find-and-apply-designers